share/completions/diskutil.fish

```diff
@@ -1,5 +1,27 @@
 # completion for diskutil (macOS)
 
+#############
+# Functions #
+#############
+
+function __fish_diskutil_needs_command
+  set cmd (commandline -opc)
+  if test (count $cmd) -eq 1 -a $cmd[1] = 'diskutil'
+    return 0
+  end
+  return 1
+end
+
+function __fish_diskutil_using_command
+  set cmd (commandline -opc)
+  if test (count $cmd) -gt 1
+    if test $argv[1] = $cmd[2]
```

Comment:
This is actually more strict than the previous `__fish_seen_subcommand_from` (which just checks if _one_ of the tokens is one of the arguments).

It'll break as soon as you have an option before a subcommand - e.g. `diskutil -plist list <TAB>` won't offer devices.

If you're not completely certain how a command parses stuff, you should err on the side of permitting things.

Reply:
Also, it's additional code. Please remove again!